src/net/serialization.cc

```diff
@@ -0,0 +1,41 @@
+/* -*-mode:c++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+
+#include "serialization.hh"
+
+using namespace std;
+
+string put_field( const uint16_t n )
+{
+  const uint16_t network_order = htobe16(n);
```

Comment:
once you decide to put spaces within `( )`, then you need to put them everywhere else (e.g., CS140 will deduct points from your assignments for not being consistent); or you could simply remove all the spaces :-)